substrate/client/network/src/bitswap/mod.rs

```diff
@@ -60,6 +60,11 @@ const MAX_WANTED_BLOCKS: usize = 16;
 /// Bitswap protocol name
 const PROTOCOL_NAME: &'static str = "/ipfs/bitswap/1.2.0";
 
+/// Check if a CID is supported by the bitswap protocol.
+pub fn is_cid_supported(cid: &Cid) -> bool {
+	cid.version() == Version::V1 && cid.hash().size() == 32
```

Comment:
@dmitry-markin wdyt?
```suggestion
	cid.version() != Version::V0 && cid.hash().size() == 32
```